frontend/src/app/features/work-packages/components/wp-reminder-modal/wp-reminder.modal.ts

```diff
@@ -57,12 +59,13 @@ export class WorkPackageReminderModalComponent extends OpModalComponent implemen
     super(locals, cdRef, elementRef);
 
     this.workPackage = this.locals.workPackage as WorkPackageResource;
+    this.preset = this.locals.preset as ReminderPreset | undefined;
     this.title$ = this
       .isEditMode()
       .pipe(
         map((isEditMode) => (isEditMode ? this.text.edit_title : this.text.new_title)),
       );
-    this.frameSrc = this.pathHelper.workPackageReminderModalBodyPath(this.workPackage.id as string);
+    this.frameSrc = this.pathHelper.workPackageReminderModalBodyPath(this.workPackage.id as string) + (this.preset ? `?preset=${this.preset}` : '');
```

Comment:
I am not a fan of building the url via string concatenation,altough it works of course, it is more personal preference. I'd prefer it being build with URL helpers like it is done [here](https://github.com/opf/openproject/blob/dev/frontend/src/app/shared/components/fields/edit/field-types/progress-popover-edit-field.component.ts#L151).